api/apitypes/apitypes.go

```diff
@@ -50,3 +50,12 @@ type Usage struct {
 func UsageByFeature(a, b Usage) bool {
 	return a.Feature.Less(b.Feature)
 }
+
+type PushResult struct {
+	Feature refs.FeaturePlan
+	Err     string
+}
+
+type PushResponse struct {
+	Errors []PushResult
```

Comment:
Can I get some lowercase json tags on these?

Reply:
Of course!